resources/user-agents/apps/microsoft/bing-search-app-generic.json

```diff
@@ -0,0 +1,95 @@
+{
+  "division": "Bing Search App Generic",
+  "sortIndex": 482,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Bing Search App Generic for Android",
+      "engine": "Blink",
+      "device": "general Mobile Phone (Touch)",
+      "platform": "Android",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Bing Search App Generic for Android",
+        "Browser": "Bing Search App",
+        "Browser_Type": "Application",
+        "Browser_Maker": "Microsoft Corporation",
+        "Crawler": "false"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* BingWeb/*",
+          "platforms": [
+            "Android_7_0",
+            "Android_6_0",
+            "Android_5_1", "Android_5_0",
+            "Android_4_4", "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0",
+            "Android"
+          ]
+        },
+        {
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/* Mobile Safari/* BingWeb/*",
+          "platforms": [
+            "Android"
```

Comment:
Why no specific Android versions here?

Reply:
The only UA I found, matching this rule, is in Test `issue-1387-DL`. Its with Andoid version and without any device information.